readme.md

```diff
@@ -1,3 +1,24 @@
 # Product Owner Guide
 
 A rough guide for those who are product owners on a dwyl project.
+
+### What is Technical Debt?
+
+Technical debt is a term to describe the implied cost of rework caused by
```

Comment:
Technical debt is the cost to "re-work" a feature (_or entire app_) because it was rushed the _first_ time it was built.
Think of Technical Debt like a "loan shark" for your project, if you take "shortcuts" to implement a feature quickly (_e.g: by skipping tests_) you will pay for it later and usually with "interest".